packages/components/scripts/generateSchema.js

```diff
@@ -0,0 +1,39 @@
+const {
+  schema,
+  DividerSchema,
+  ParagraphSchema,
+  OrderedListSchema,
+  UnorderedListSchema,
+  ProseSchema,
+} = require("@opengovsg/isomer-components")
+const fs = require("fs")
+const path = require("path")
+
+/**
+ * This script generates a JSON schema file for the Isomer JSON Schema. It is
+ * kept in the docs/next folder so that POEs can use the schema in third-party
+ * form builders when migrating content to Isomer Next.
+ */
+
+const jsonOutput = JSON.stringify(
+  {
+    ...schema,
+    components: {
```

Comment:
earlier have segmented this components key furthers in complex native internal right? have we removed that?

Reply:
The segmentation is still maintained in Studio via `IsomerComplexComponentsMap` and `IsomerNativeComponentsMap`! This schema is generated this way to fit into RJSF's form builder, in which they are only able to resolve `$ref`s using paths instead of `$id`.

This functionality is not needed inside JSONForms because `$ref`s are resolved using `$id`s.